src/events.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Dispatches event named onFeedback
+ * Implement an addEventListener at the event target to utilise
+ *
+ * @param {HTMLElement} container DOM node to define event target
+ * @param {boolean} satisfied Bool to send user satisfaction
+ */
+export default function OnFeedbackEvent(container, satisfied) {
```

Comment:
Function names should not start with a capital letter. Only classes should be capitalised.
I'm surprised eslint didn't complain about this 🤔

Reply:
Thought it looked off but blame this thread for throwing me https://stackoverflow.com/questions/48211891/import-functions-from-another-js-file-javascript

Reply:
Yeah that's an old-skool method of creating classes in javascript before the `class` keyword existed.

Technically you can do:
```
function MyClass() {
}
var instance = new MyClass();
```